test/locations.service.integration.js

```diff
@@ -17,7 +17,7 @@ function shouldNotBeCalled (rejection) {
 }
 
 describe('locationsService', function () {
-  var url = window.__env__.COUCHDB_URL || 'http://localhost:5984'
```

Comment:
Was this causing any issues? I added it for ease of testing locally (can just run `npm t`)